iothub/device/src/GlobalSuppressions.cs

```diff
@@ -11,3 +11,4 @@
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "SDK hides non-actionable errors from user", Scope = "module")]
 [assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Public API cannot be changed.", Scope = "type", Target = "~T:Microsoft.Azure.Devices.Client.ConnectionStatusChangeReason")]
```

Comment:
We should probably keep code-specific suppressions next to where the code is, and save this file for module-wide suppressions.

Reply:
sure